ur_robot_driver/doc/ROS_INTERFACE.md

```diff
@@ -26,7 +26,7 @@ Debug flag that will get passed on to ur_common.launch
 
 Automatically send URScript to robot to execute. On e-Series this does require the robot to be in 'remote-control' mode. With this, the URCap is not needed on the robot.
 
-##### kinematics_config (default: "$(find ur_e_description)/config/ur3e_default.yaml")
+##### kinematics_config (default: "$(find ur_description)/config/ur3e/default_kinematics.yaml")
 
 Kinematics config file used for calibration correction. This will be used to verify the robot's calibration is matching the robot_description.
```

Comment:
Suggestion: be vocal about the fact that the `default` value will *not* result in accurate FK results -- as it's really just a default, with the values of a "random robot".

Reply:
Added a note in 00b09ba